tutorials/fundamentals/check-date-between-two-dates.md

```diff
@@ -0,0 +1,28 @@
+You can check if a date is between two dates by simply using the `>=` and `<=` operators.
+
+```javascript
+const start = Date.parse('04 Dec 1995 00:12:00 GMT');
+const end = Date.now();
+const d = Date.parse('24 Dec 1997 13:47:00');
+
+if (d >= start && d <= end) {
+  console.log('This date is between the two dates');
+} else {
+  console.log('This date is not between the two dates');
+}
+```
+
+Typescript doesn't like this approach and will complain.
+To get Typescript to shut up, use the `valueOf()` function in conjunction with `Date()`.
```

Comment:
Lol 😂 probably not good to put shut up here. More like "To make TypeScript happy, use valueOf()"